src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -88,6 +97,9 @@ const getId = ({ pageType, service, variant, env, isCaf }: GetIdProps) => {
     case LIVE_PAGE:
```

Comment:
I think you can maybe get rid of passing around the `isArchive` flag and do the URL checking under the `case LIVE_PAGE` case, maybe something like:

```
getIdFunction = (path: string) => {
        if (!path.match(/(c[a-zA-Z0-9]{10}t)/)) {
          const cpsId = getCpsId(path);
          return `/${service}/live/${cpsId}`;
        }
        return getTipoId(path);
      };
```

That would check that if the ID is _not_ a TIPO ID structure, then assume CPS, or else return the TIPO ID.

It doesn't add the `renderer_env` flag though. The `[[...variant]].page.tsx` file would need amended to set the `rendererEnv` prop as a default of `test`, which would then let us override it with `renderer_env=live` if we want to test CPS live pages.

Reply:
oh wow yea this is much simpler thank you! will implement now 🥳